### ISSUE TITLE
max-statements-per-line not reported about arrow functions

### ISSUE THREAD

**Issue by evilebottnawi** [eslint/eslint#7784]
**Tell us about your environment**

* **ESLint Version:** `3.12.2`
* **Node Version:** `7.2.1`
* **npm Version:** `3.10.10`

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**

```json
{
    "rules": {
        "max-statements-per-line": ["error", 1]
    }
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
module.exports = (options) => (callback) => { 
    console.log(options);

    return callback()
}
```

**What did you expect to happen?**

Error.

```js
module.exports = (options) => 
    (callback) => { 
        console.log(options);

        return callback()
    }
```

**What actually happened? Please include the actual, raw output from ESLint.**

Nothing

**Comment by not-an-aardvark** [comment]
Hi, thanks for the report.

However, I think this is working as intended. There are multiple arrow functions on the first line, but there is only one statement. This is similar to the following code:

```js
module.exports = (options) => foo.bar.baz();
```

**Comment by evilebottnawi** [comment]
@not-an-aardvark thanks for answer, maybe good create own rule for this purpose
